### ISSUE TITLE
Unable to re-arrange "Open in New Container Tab" container list

### ISSUE THREAD

**Issue by wumpus42** [mozilla/multi-account-containers#2399]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

0. Install Firefox Multi-Account Containers add-on
1. Choose a link from the current webpage, Bookmarks Sidebar, Bookmarks Toolbar, etc.
2. Right-click link
3. Navigate to the "Open in New Container Tab" option
4. Move pointer to reveal current container list
5. How do I re-arrange that list?

### Actual behavior

The list is static. I thought there was a way to re-arrange it previously?

### Expected behavior

Make the list re-arrange-able.

### Provide a copy of Troubleshooting Information page

```plain text
Application Basics
------------------

Name: Firefox
Version: 103.0
Build ID: 20220718155818
Distribution ID:
Update Channel: release
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:103.0) Gecko/20100101 Firefox/103.0
OS: Windows_NT 10.0 19045
Launcher Process: Enabled
Multiprocess Windows: 1/1
Fission Windows: 1/1 Enabled by default
Remote Processes: 17
Enterprise Policies: Inactive
Google Location Service Key: Found
Google Safebrowsing Key: Found
Mozilla Location Service Key: Found
Safe Mode: false
Memory Size (RAM): 8.0 GB
Disk Space Available: 362 GB

Crash Reports for the Last 3 Days
---------------------------------

Firefox Features
----------------

Name: DoH Roll-Out
Version: 2.0.0
ID: doh-rollout@mozilla.org

Name: Firefox Screenshots
Version: 39.0.1
ID: screenshots@mozilla.org

Name: Form Autofill
Version: 1.0.1
ID: formautofill@mozilla.org

Name: Picture-In-Picture
Version: 1.0.0
ID: pictureinpicture@mozilla.org

Name: Web Compatibility Interventions
Version: 103.0.0
ID: webcompat@mozilla.org

Name: WebCompat Reporter
Version: 1.5.0
ID: webcompat-reporter@mozilla.org

Remote Features
---------------

bug-1693420-rollout-sponsored-top-sites-rollout-release-84-100: active
bug-1712189-rollout-keep-firefox-up-to-date-even-when-it-is-not-ru-release-90-94: active
bug-1766468-rollout-win32k-rollout-release-100-101: active
bug-1766618-rollout-disable-websocket-over-http2-release-91-100: active

Remote Processes
----------------

Type: Privileged About
Count: 1

Type: Extension
Count: 1

Type: Isolated Web Content
Count: 10

Type: Preallocated
Count: 3

Type: GPU
Count: 1

Type: Socket
Count: 1

Add-ons
-------

Name: Add-ons Search Detection
Type: extension
Version: 2.0.0
Enabled: true
ID: addons-search-detection@mozilla.com

Name: Amazon.com
Type: extension
Version: 1.3
Enabled: true
ID: amazondotcom@search.mozilla.org

Name: Bing
Type: extension
Version: 1.3
Enabled: true
ID: bing@search.mozilla.org

Name: Copyfish ?? Free OCR Software
Type: extension
Version: 5.1.9
Enabled: true
ID: copyfish@a9t9.com

Name: DuckDuckGo
Type: extension
Version: 1.1
Enabled: true
ID: ddg@search.mozilla.org

Name: eBay
Type: extension
Version: 1.3
Enabled: true
ID: ebay@search.mozilla.org

Name: Facebook Container
Type: extension
Version: 2.3.2
Enabled: true
ID: @contain-facebook

Name: Firefox Multi-Account Containers
Type: extension
Version: 8.0.7
Enabled: true
ID: @testpilot-containers

Name: Google
Type: extension
Version: 1.2
Enabled: true
ID: google@search.mozilla.org

Name: Nuke Anything
Type: extension
Version: 2.4
Enabled: true
ID: {1ced4832-f06e-413f-aa14-9eb63ad40ace}

Name: Privacy Badger
Type: extension
Version: 2021.11.23.1
Enabled: true
ID: jid1-MnnxcxisBPnSXQ@jetpack

Name: uBlock Origin
Type: extension
Version: 1.43.0
Enabled: true
ID: uBlock0@raymondhill.net

Name: Video DownloadHelper
Type: extension
Version: 7.6.0
Enabled: true
ID: {b9db16a4-6edc-47ec-a1f4-b86292ed211d}

Name: Wikipedia (en)
Type: extension
Version: 1.1
Enabled: true
ID: wikipedia@search.mozilla.org

Security Software
-----------------

Type: Microsoft Defender Antivirus

Type:

Type: Windows Firewall

Graphics
--------

Features
Compositing: WebRender
Asynchronous Pan/Zoom: wheel input enabled; scrollbar drag enabled; keyboard enabled; autoscroll enabled; smooth pinch-zoom enabled
WebGL 1 Driver WSI Info: EGL_VENDOR: Google Inc. (NVIDIA) EGL_VERSION: 1.5 (ANGLE 2.1.15727 git hash: 23851a53779d) EGL_EXTENSIONS: EGL_EXT_create_context_robustness EGL_ANGLE_d3d_share_handle_client_buffer EGL_ANGLE_d3d_texture_client_buffer EGL_ANGLE_surface_d3d_texture_2d_share_handle EGL_ANGLE_query_surface_pointer EGL_ANGLE_window_fixed_size EGL_ANGLE_keyed_mutex EGL_ANGLE_surface_orientation EGL_ANGLE_direct_composition EGL_NV_post_sub_buffer EGL_KHR_create_context EGL_KHR_image EGL_KHR_image_base EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_gl_renderbuffer_image EGL_KHR_get_all_proc_addresses EGL_KHR_stream EGL_KHR_stream_consumer_gltexture EGL_NV_stream_consumer_gltexture_yuv EGL_ANGLE_flexible_surface_compatibility EGL_ANGLE_stream_producer_d3d_texture EGL_ANGLE_create_context_webgl_compatibility EGL_CHROMIUM_create_context_bind_generates_resource EGL_CHROMIUM_sync_control EGL_EXT_pixel_format_float EGL_KHR_surfaceless_context EGL_ANGLE_display_texture_share_group EGL_ANGLE_display_semaphore_share_group EGL_ANGLE_create_context_client_arrays EGL_ANGLE_program_cache_control EGL_ANGLE_robust_resource_initialization EGL_ANGLE_create_context_extensions_enabled EGL_ANDROID_blob_cache EGL_ANDROID_recordable EGL_ANGLE_image_d3d11_texture EGL_ANGLE_create_context_backwards_compatible EGL_KHR_create_context_no_error EGL_KHR_reusable_sync EGL_EXTENSIONS(nullptr): EGL_EXT_client_extensions EGL_EXT_device_query EGL_EXT_platform_base EGL_EXT_platform_device EGL_ANGLE_platform_angle EGL_ANGLE_platform_angle_d3d EGL_ANGLE_platform_angle_d3d11on12 EGL_ANGLE_device_creation EGL_ANGLE_device_creation_d3d11 EGL_ANGLE_experimental_present_path EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_ANGLE_explicit_context EGL_ANGLE_feature_control IsWebglOutOfProcessEnabled: 1
WebGL 1 Driver Renderer: Google Inc. (NVIDIA) -- ANGLE (NVIDIA, NVIDIA GeForce GT 750M Direct3D11 vs_5_0 ps_5_0, D3D11-25.21.14.1917)
WebGL 1 Driver Version: OpenGL ES 2.0.0 (ANGLE 2.1.15727 git hash: 23851a53779d)
WebGL 1 Driver Extensions: GL_ANGLE_base_vertex_base_instance GL_ANGLE_client_arrays GL_ANGLE_depth_texture GL_ANGLE_explicit_context GL_ANGLE_explicit_context_gles1 GL_ANGLE_framebuffer_blit GL_ANGLE_framebuffer_multisample GL_ANGLE_get_serialized_context_string GL_ANGLE_get_tex_level_parameter GL_ANGLE_instanced_arrays GL_ANGLE_lossy_etc_decode GL_ANGLE_memory_size GL_ANGLE_multi_draw GL_ANGLE_multiview_multisample GL_ANGLE_pack_reverse_row_order GL_ANGLE_program_cache_control GL_ANGLE_provoking_vertex GL_ANGLE_request_extension GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ANGLE_texture_usage GL_ANGLE_translated_shader_source GL_CHROMIUM_bind_generates_resource GL_CHROMIUM_bind_uniform_location GL_CHROMIUM_color_buffer_float_rgb GL_CHROMIUM_color_buffer_float_rgba GL_CHROMIUM_copy_compressed_texture GL_CHROMIUM_copy_texture GL_CHROMIUM_lose_context GL_CHROMIUM_sync_query GL_EXT_EGL_image_external_wrap_modes GL_EXT_blend_func_extended GL_EXT_blend_minmax GL_EXT_color_buffer_half_float GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_disjoint_timer_query GL_EXT_draw_buffers GL_EXT_draw_elements_base_vertex GL_EXT_float_blend GL_EXT_frag_depth GL_EXT_instanced_arrays GL_EXT_map_buffer_range GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_sRGB GL_EXT_shader_texture_lod GL_EXT_texture_compression_bptc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc_srgb GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_rg GL_EXT_texture_storage GL_EXT_unpack_subimage GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_NV_EGL_stream_consumer_external GL_NV_fence GL_NV_pack_subimage GL_NV_pixel_buffer_object GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_compressed_EAC_R11_signed_texture GL_OES_compressed_EAC_R11_unsigned_texture GL_OES_compressed_EAC_RG11_signed_texture GL_OES_compressed_EAC_RG11_unsigned_texture GL_OES_compressed_ETC2_RGB8_texture GL_OES_compressed_ETC2_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture GL_OES_compressed_ETC2_sRGB8_alpha8_texture GL_OES_compressed_ETC2_sRGB8_texture GL_OES_depth24 GL_OES_depth32 GL_OES_draw_elements_base_vertex GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_get_program_binary GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_texture_border_clamp GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_vertex_array_object GL_WEBGL_video_texture
WebGL 1 Extensions: ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_float_blend EXT_frag_depth EXT_shader_texture_lod EXT_sRGB EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_element_index_uint OES_fbo_render_mipmap OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context
WebGL 2 Driver WSI Info: EGL_VENDOR: Google Inc. (NVIDIA) EGL_VERSION: 1.5 (ANGLE 2.1.15727 git hash: 23851a53779d) EGL_EXTENSIONS: EGL_EXT_create_context_robustness EGL_ANGLE_d3d_share_handle_client_buffer EGL_ANGLE_d3d_texture_client_buffer EGL_ANGLE_surface_d3d_texture_2d_share_handle EGL_ANGLE_query_surface_pointer EGL_ANGLE_window_fixed_size EGL_ANGLE_keyed_mutex EGL_ANGLE_surface_orientation EGL_ANGLE_direct_composition EGL_NV_post_sub_buffer EGL_KHR_create_context EGL_KHR_image EGL_KHR_image_base EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_gl_renderbuffer_image EGL_KHR_get_all_proc_addresses EGL_KHR_stream EGL_KHR_stream_consumer_gltexture EGL_NV_stream_consumer_gltexture_yuv EGL_ANGLE_flexible_surface_compatibility EGL_ANGLE_stream_producer_d3d_texture EGL_ANGLE_create_context_webgl_compatibility EGL_CHROMIUM_create_context_bind_generates_resource EGL_CHROMIUM_sync_control EGL_EXT_pixel_format_float EGL_KHR_surfaceless_context EGL_ANGLE_display_texture_share_group EGL_ANGLE_display_semaphore_share_group EGL_ANGLE_create_context_client_arrays EGL_ANGLE_program_cache_control EGL_ANGLE_robust_resource_initialization EGL_ANGLE_create_context_extensions_enabled EGL_ANDROID_blob_cache EGL_ANDROID_recordable EGL_ANGLE_image_d3d11_texture EGL_ANGLE_create_context_backwards_compatible EGL_KHR_create_context_no_error EGL_KHR_reusable_sync EGL_EXTENSIONS(nullptr): EGL_EXT_client_extensions EGL_EXT_device_query EGL_EXT_platform_base EGL_EXT_platform_device EGL_ANGLE_platform_angle EGL_ANGLE_platform_angle_d3d EGL_ANGLE_platform_angle_d3d11on12 EGL_ANGLE_device_creation EGL_ANGLE_device_creation_d3d11 EGL_ANGLE_experimental_present_path EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_ANGLE_explicit_context EGL_ANGLE_feature_control IsWebglOutOfProcessEnabled: 1
WebGL 2 Driver Renderer: Google Inc. (NVIDIA) -- ANGLE (NVIDIA, NVIDIA GeForce GT 750M Direct3D11 vs_5_0 ps_5_0, D3D11-25.21.14.1917)
WebGL 2 Driver Version: OpenGL ES 3.0.0 (ANGLE 2.1.15727 git hash: 23851a53779d)
WebGL 2 Driver Extensions: GL_ANGLE_base_vertex_base_instance GL_ANGLE_client_arrays GL_ANGLE_copy_texture_3d GL_ANGLE_depth_texture GL_ANGLE_explicit_context GL_ANGLE_explicit_context_gles1 GL_ANGLE_framebuffer_blit GL_ANGLE_framebuffer_multisample GL_ANGLE_get_serialized_context_string GL_ANGLE_get_tex_level_parameter GL_ANGLE_instanced_arrays GL_ANGLE_lossy_etc_decode GL_ANGLE_memory_size GL_ANGLE_multi_draw GL_ANGLE_multiview_multisample GL_ANGLE_pack_reverse_row_order GL_ANGLE_program_cache_control GL_ANGLE_provoking_vertex GL_ANGLE_request_extension GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ANGLE_texture_multisample GL_ANGLE_texture_usage GL_ANGLE_translated_shader_source GL_CHROMIUM_bind_generates_resource GL_CHROMIUM_bind_uniform_location GL_CHROMIUM_copy_compressed_texture GL_CHROMIUM_copy_texture GL_CHROMIUM_lose_context GL_CHROMIUM_sync_query GL_EXT_EGL_image_external_wrap_modes GL_EXT_blend_func_extended GL_EXT_blend_minmax GL_EXT_color_buffer_float GL_EXT_color_buffer_half_float GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_disjoint_timer_query GL_EXT_draw_buffers GL_EXT_draw_buffers_indexed GL_EXT_draw_elements_base_vertex GL_EXT_float_blend GL_EXT_frag_depth GL_EXT_instanced_arrays GL_EXT_map_buffer_range GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_sRGB GL_EXT_shader_texture_lod GL_EXT_texture_compression_bptc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc_srgb GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_norm16 GL_EXT_texture_rg GL_EXT_texture_storage GL_EXT_unpack_subimage GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_NV_EGL_stream_consumer_external GL_NV_fence GL_NV_pack_subimage GL_NV_pixel_buffer_object GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_EGL_image_external_essl3 GL_OES_compressed_EAC_R11_signed_texture GL_OES_compressed_EAC_R11_unsigned_texture GL_OES_compressed_EAC_RG11_signed_texture GL_OES_compressed_EAC_RG11_unsigned_texture GL_OES_compressed_ETC2_RGB8_texture GL_OES_compressed_ETC2_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture GL_OES_compressed_ETC2_sRGB8_alpha8_texture GL_OES_compressed_ETC2_sRGB8_texture GL_OES_depth24 GL_OES_depth32 GL_OES_draw_buffers_indexed GL_OES_draw_elements_base_vertex GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_get_program_binary GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_texture_border_clamp GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_vertex_array_object GL_OVR_multiview GL_OVR_multiview2 GL_WEBGL_video_texture
WebGL 2 Extensions: EXT_color_buffer_float EXT_float_blend EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_texture_float_linear OVR_multiview2 WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context
Direct2D: true
Target Frame Rate: 60
DirectWrite: true (10.0.19041.1566)
GPU #1
Active: Yes
Description: NVIDIA GeForce GT 750M
Vendor ID: 0x10de
Device ID: 0x0fe4
Driver Version: 25.21.14.1917
Driver Date: 2-20-2019
Drivers: C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumdx.dll,C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumdx.dll,C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumdx.dll,C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumdx.dll C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumd.dll,C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumd.dll,C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumd.dll,C:\Windows\System32\DriverStore\FileRepository\nvlti.inf_amd64_3a7de86f44009d24\nvldumd.dll
Subsys ID: 380017aa
RAM: 2048
GPU #2
Active: No
RAM: 0
Diagnostics
AzureCanvasBackend: direct2d 1.1
AzureCanvasBackend (UI Process): skia
AzureContentBackend: skia
AzureContentBackend (UI Process): skia
AzureFallbackCanvasBackend (UI Process): skia
CMSOutputProfile: AAAMSExpbm8CEAAAbW50clJHQiBYWVogB84AAgAJAAYAMQAAYWNzcE1TRlQAAAAASUVDIHNSR0IAAAAAAAAAAAAAAAAAAPbWAAEAAAAA0y1IUCAgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAARY3BydAAAAVAAAAAzZGVzYwAAAYQAAABsd3RwdAAAAfAAAAAUYmtwdAAAAgQAAAAUclhZWgAAAhgAAAAUZ1hZWgAAAiwAAAAUYlhZWgAAAkAAAAAUZG1uZAAAAlQAAABwZG1kZAAAAsQAAACIdnVlZAAAA0wAAACGdmlldwAAA9QAAAAkbHVtaQAAA/gAAAAUbWVhcwAABAwAAAAkdGVjaAAABDAAAAAMclRSQwAABDwAAAgMZ1RSQwAABDwAAAgMYlRSQwAABDwAAAgMdGV4dAAAAABDb3B5cmlnaHQgKGMpIDE5OTggSGV3bGV0dC1QYWNrYXJkIENvbXBhbnkAAGRlc2MAAAAAAAAAEnNSR0IgSUVDNjE5NjYtMi4xAAAAAAAAAAAAAAASc1JHQiBJRUM2MTk2Ni0yLjEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAFhZWiAAAAAAAADzUQABAAAAARbMWFlaIAAAAAAAAAAAAAAAAAAAAABYWVogAAAAAAAAb6IAADj1AAADkFhZWiAAAAAAAABimQAAt4UAABjaWFlaIAAAAAAAACSgAAAPhAAAts9kZXNjAAAAAAAAABZJRUMgaHR0cDovL3d3dy5pZWMuY2gAAAAAAAAAAAAAABZJRUMgaHR0cDovL3d3dy5pZWMuY2gAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAZGVzYwAAAAAAAAAuSUVDIDYxOTY2LTIuMSBEZWZhdWx0IFJHQiBjb2xvdXIgc3BhY2UgLSBzUkdCAAAAAAAAAAAAAAAuSUVDIDYxOTY2LTIuMSBEZWZhdWx0IFJHQiBjb2xvdXIgc3BhY2UgLSBzUkdCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGRlc2MAAAAAAAAALFJlZmVyZW5jZSBWaWV3aW5nIENvbmRpdGlvbiBpbiBJRUM2MTk2Ni0yLjEAAAAAAAAAAAAAACxSZWZlcmVuY2UgVmlld2luZyBDb25kaXRpb24gaW4gSUVDNjE5NjYtMi4xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB2aWV3AAAAAAATpP4AFF8uABDPFAAD7cwABBMLAANcngAAAAFYWVogAAAAAABMCVYAUAAAAFcf521lYXMAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAKPAAAAAnNpZyAAAAAAQ1JUIGN1cnYAAAAAAAAEAAAAAAUACgAPABQAGQAeACMAKAAtADIANwA7AEAARQBKAE8AVABZAF4AYwBoAG0AcgB3AHwAgQCGAIsAkACVAJoAnwCkAKkArgCyALcAvADBAMYAywDQANUA2wDgAOUA6wDwAPYA+wEBAQcBDQETARkBHwElASsBMgE4AT4BRQFMAVIBWQFgAWcBbgF1AXwBgwGLAZIBmgGhAakBsQG5AcEByQHRAdkB4QHpAfIB+gIDAgwCFAIdAiYCLwI4AkECSwJUAl0CZwJxAnoChAKOApgCogKsArYCwQLLAtUC4ALrAvUDAAMLAxYDIQMtAzgDQwNPA1oDZgNyA34DigOWA6IDrgO6A8cD0wPgA+wD+QQGBBMEIAQtBDsESARVBGMEcQR+BIwEmgSoBLYExATTBOEE8AT+BQ0FHAUrBToFSQVYBWcFdwWGBZYFpgW1BcUF1QXlBfYGBgYWBicGNwZIBlkGagZ7BowGnQavBsAG0QbjBvUHBwcZBysHPQdPB2EHdAeGB5kHrAe/B9IH5Qf4CAsIHwgyCEYIWghuCIIIlgiqCL4I0gjnCPsJEAklCToJTwlkCXkJjwmkCboJzwnlCfsKEQonCj0KVApqCoEKmAquCsUK3ArzCwsLIgs5C1ELaQuAC5gLsAvIC+EL+QwSDCoMQwxcDHUMjgynDMAM2QzzDQ0NJg1ADVoNdA2ODakNww3eDfgOEw4uDkkOZA5/DpsOtg7SDu4PCQ8lD0EPXg96D5YPsw/PD+wQCRAmEEMQYRB+EJsQuRDXEPURExExEU8RbRGMEaoRyRHoEgcSJhJFEmQShBKjEsMS4xMDEyMTQxNjE4MTpBPFE+UUBhQnFEkUahSLFK0UzhTwFRIVNBVWFXgVmxW9FeAWAxYmFkkWbBaPFrIW1hb6Fx0XQRdlF4kXrhfSF/cYGxhAGGUYihivGNUY+hkgGUUZaxmRGbcZ3RoEGioaURp3Gp4axRrsGxQbOxtjG4obshvaHAIcKhxSHHscoxzMHPUdHh1HHXAdmR3DHeweFh5AHmoelB6+HukfEx8+H2kflB+/H+ogFSBBIGwgmCDEIPAhHCFIIXUhoSHOIfsiJyJVIoIiryLdIwojOCNmI5QjwiPwJB8kTSR8JKsk2iUJJTglaCWXJccl9yYnJlcmhya3JugnGCdJJ3onqyfcKA0oPyhxKKIo1CkGKTgpaymdKdAqAio1KmgqmyrPKwIrNitpK50r0SwFLDksbiyiLNctDC1BLXYtqy3hLhYuTC6CLrcu7i8kL1ovkS/HL/4wNTBsMKQw2zESMUoxgjG6MfIyKjJjMpsy1DMNM0YzfzO4M/E0KzRlNJ402DUTNU01hzXCNf02NzZyNq426TckN2A3nDfXOBQ4UDiMOMg5BTlCOX85vDn5OjY6dDqyOu87LTtrO6o76DwnPGU8pDzjPSI9YT2hPeA+ID5gPqA+4D8hP2E/oj/iQCNAZECmQOdBKUFqQaxB7kIwQnJCtUL3QzpDfUPARANER0SKRM5FEkVVRZpF3kYiRmdGq0bwRzVHe0fASAVIS0iRSNdJHUljSalJ8Eo3Sn1KxEsMS1NLmkviTCpMcky6TQJNSk2TTdxOJU5uTrdPAE9JT5NP3VAnUHFQu1EGUVBRm1HmUjFSfFLHUxNTX1OqU/ZUQlSPVNtVKFV1VcJWD1ZcVqlW91dEV5JX4FgvWH1Yy1kaWWlZuFoHWlZaplr1W0VblVvlXDVchlzWXSddeF3JXhpebF69Xw9fYV+zYAVgV2CqYPxhT2GiYfViSWKcYvBjQ2OXY+tkQGSUZOllPWWSZedmPWaSZuhnPWeTZ+loP2iWaOxpQ2maafFqSGqfavdrT2una/9sV2yvbQhtYG25bhJua27Ebx5veG/RcCtwhnDgcTpxlXHwcktypnMBc11zuHQUdHB0zHUodYV14XY+dpt2+HdWd7N4EXhueMx5KnmJeed6RnqlewR7Y3vCfCF8gXzhfUF9oX4BfmJ+wn8jf4R/5YBHgKiBCoFrgc2CMIKSgvSDV4O6hB2EgITjhUeFq4YOhnKG14c7h5+IBIhpiM6JM4mZif6KZIrKizCLlov8jGOMyo0xjZiN/45mjs6PNo+ekAaQbpDWkT+RqJIRknqS45NNk7aUIJSKlPSVX5XJljSWn5cKl3WX4JhMmLiZJJmQmfyaaJrVm0Kbr5wcnImc951kndKeQJ6unx2fi5/6oGmg2KFHobaiJqKWowajdqPmpFakx6U4pammGqaLpv2nbqfgqFKoxKk3qamqHKqPqwKrdavprFys0K1ErbiuLa6hrxavi7AAsHWw6rFgsdayS7LCszizrrQltJy1E7WKtgG2ebbwt2i34LhZuNG5SrnCuju6tbsuu6e8IbybvRW9j74KvoS+/796v/XAcMDswWfB48JfwtvDWMPUxFHEzsVLxcjGRsbDx0HHv8g9yLzJOsm5yjjKt8s2y7bMNcy1zTXNtc42zrbPN8+40DnQutE80b7SP9LB00TTxtRJ1MvVTtXR1lXW2Ndc1+DYZNjo2WzZ8dp22vvbgNwF3IrdEN2W3hzeot8p36/gNuC94UThzOJT4tvjY+Pr5HPk/OWE5g3mlucf56noMui86Ubp0Opb6uXrcOv77IbtEe2c7ijutO9A78zwWPDl8XLx//KM8xnzp/Q09ML1UPXe9m32+/eK+Bn4qPk4+cf6V/rn+3f8B/yY/Sn9uv5L/tz/bf//
Display0: 1920x1080@60Hz scales:1.000000|1.000000
Display1: 1920x1080@60Hz scales:1.000000|1.000000
DisplayCount: 2
HardwareStretching: both=0 window-only=0 full-screen-only=2 none=0 error=0
GPUProcessPid: 35452
ClearType Parameters: \\.\DISPLAY1 [ Gamma: 1.8 Pixel Structure: RGB ClearType Level: 100 Enhanced Contrast: 50 ] \\.\DISPLAY2 [ Gamma: 1.8 Pixel Structure: RGB ClearType Level: 100 Enhanced Contrast: 50 ]
Decision Log
HW_COMPOSITING:
available by default
D3D11_COMPOSITING:
available by default
DIRECT2D:
available by default
D3D11_HW_ANGLE:
available by default
GPU_PROCESS:
available by default
WEBRENDER:
available by default
WEBRENDER_QUALIFIED:
available by default
WEBRENDER_COMPOSITOR:
available by default
WEBRENDER_PARTIAL:
available by default
WEBRENDER_SHADER_CACHE:
available by default
WEBRENDER_OPTIMIZED_SHADERS:
available by default
WEBRENDER_ANGLE:
available by default
WEBRENDER_DCOMP_PRESENT:
available by default
WEBRENDER_SOFTWARE:
available by default
WEBGPU:
disabled by default: Disabled by default
blocked by runtime: WebGPU cannot be enabled in release or beta
WINDOW_OCCLUSION:
available by default
VIDEO_OVERLAY:
blocked by default: Blocklisted by gfxInfo
HW_DECODED_VIDEO_ZERO_COPY:
blocked by default: Blocklisted by gfxInfo
VP8_HW_DECODE:
available by default
VP9_HW_DECODE:
available by default
REUSE_DECODER_DEVICE:
blocked by default: Blocklisted by gfxInfo




Media
-----

Audio Backend: wasapi
Max Channels: 2
Preferred Sample Rate: 44100
Roundtrip latency (standard deviation): ...
Output Devices
Name: Group
Speakers (Realtek High Definition Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0282&SUBSYS_17AA1002&REV_1000\4&3872eb08&0&0001
VE247 (High Definition Audio Device): HDAUDIO\FUNC_01&VEN_10DE&DEV_0042&SUBSYS_17AA3978&REV_1001\5&12175013&0&0001
Headphones (YaRight’s Bose headphones Stereo): BTHENUM\{0000110b-0000-1000-8000-00805f9b34fb}_VID&0001009e_PID&4020\7&ed52da7&0&2C41A1C884FA_C00000000
Headset (YaRight’s Bose headphones Hands-Free AG Audio): BTHHFENUM\BthHFPAudio\8&12518eec&0&97
Realtek Digital Output (Realtek High Definition Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0282&SUBSYS_17AA1002&REV_1000\4&3872eb08&0&0001
Input Devices
Name: Group
Microphone (Realtek High Definition Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0282&SUBSYS_17AA1002&REV_1000\4&3872eb08&0&0001
Stereo Mix (Realtek High Definition Audio): HDAUDIO\FUNC_01&VEN_10EC&DEV_0282&SUBSYS_17AA1002&REV_1000\4&3872eb08&0&0001
Headset (YaRight’s Bose headphones Hands-Free AG Audio): BTHHFENUM\BthHFPAudio\8&12518eec&0&97

Enumerate database

Environment Variables
---------------------

MOZ_CRASHREPORTER_DATA_DIRECTORY: C:\Users\YaRight\AppData\Roaming\Mozilla\Firefox\Crash Reports
MOZ_CRASHREPORTER_EVENTS_DIRECTORY: C:\Users\YaRight\AppData\Roaming\Mozilla\Firefox\Crash Reports\events
MOZ_CRASHREPORTER_PING_DIRECTORY: C:\Users\YaRight\AppData\Roaming\Mozilla\Firefox\Pending Pings
MOZ_CRASHREPORTER_RESTART_ARG_0: C:\Program Files\Mozilla\Firefox\firefox.exe
MOZ_CRASHREPORTER_STRINGS_OVERRIDE: C:\Program Files\Mozilla\Firefox\browser\crashreporter-override.ini

Experimental Features
---------------------

about:home startup cache (browser.startup.homepage.abouthome_cache.enabled): false
Cookies: SameSite=Lax by default (network.cookie.sameSite.laxByDefault): false
Cookies: SameSite=None requires secure attribute (network.cookie.sameSite.noneRequiresSecure): false
Cookies: Schemeful SameSite (network.cookie.sameSite.schemeful): false
CSS: Masonry Layout (layout.css.grid-template-masonry-value.enabled): false
Developer Tools: Compatibility Panel (devtools.inspector.compatibility.enabled): true
Developer Tools: Service Worker debugging (devtools.debugger.features.windowless-service-workers): false
Media: JPEG XL (image.jxl.enabled): false
Text Recognition (dom.text-recognition.enabled): false
Address Bar: show results during IME composition (browser.urlbar.keepPanelOpenDuringImeComposition): false
Web API: WebGPU (dom.webgpu.enabled): false
WebRTC Global Mute Toggles (privacy.webrtc.globalMuteToggles): false

Remote Experiments
------------------

Important Modified Preferences
------------------------------

accessibility.typeaheadfind.flashBar: 0
browser.contentblocking.category: strict
browser.download.useDownloadDir: false
browser.search.openintab: true
browser.search.region: US
browser.search.suggest.enabled: false
browser.sessionstore.upgradeBackup.latestBuildID: 20220718155818
browser.startup.homepage_override.buildID: 20220718155818
browser.startup.homepage_override.mstone: 103.0
browser.tabs.closeWindowWithLastTab: false
browser.urlbar.placeholderName: DuckDuckGo
browser.urlbar.placeholderName.private: DuckDuckGo
browser.urlbar.quicksuggest.migrationVersion: 2
browser.urlbar.quicksuggest.scenario: offline
browser.urlbar.resultGroups: {"children":[{"maxResultCount":1,"children":[{"group":"heuristicTest"},{"group":"heuristicExtension"},{"group":"heuristi
browser.urlbar.showSearchSuggestionsFirst: false
browser.urlbar.suggest.engines: false
browser.urlbar.suggest.openpage: false
browser.urlbar.suggest.quicksuggest.nonsponsored: false
browser.urlbar.suggest.quicksuggest.sponsored: false
browser.urlbar.suggest.searches: false
browser.urlbar.suggest.topsites: false
browser.zoom.full: false
doh-rollout.balrog-migration-done: true
doh-rollout.disable-heuristics: true
doh-rollout.doneFirstRun: true
doh-rollout.home-region: US
doh-rollout.uri: https://mozilla.cloudflare-dns.com/dns-query
dom.forms.autocomplete.formautofill: true
dom.security.https_only_mode: true
dom.security.https_only_mode_ever_enabled: true
dom.storage.next_gen_auto_enabled_by_cause1: true
extensions.formautofill.addresses.enabled: false
extensions.formautofill.creditCards.enabled: false
extensions.lastAppVersion: 103.0
font.internaluseonly.changed: true
gfx.crash-guard.status.wmfvpxvideo: 2
gfx.crash-guard.wmfvpxvideo.appVersion: 98.0.2
gfx.crash-guard.wmfvpxvideo.deviceID: 0x0fe4
gfx.crash-guard.wmfvpxvideo.driverVersion: 25.21.14.1917
idle.lastDailyNotification: 1658869931
media.autoplay.default: 5
media.gmp-gmpopenh264.abi: x86_64-msvc-x64
media.gmp-gmpopenh264.lastUpdate: 1655491715
media.gmp-gmpopenh264.version: 1.8.1.2
media.gmp-manager.buildID: 20220718155818
media.gmp-manager.lastCheck: 1658867012
media.gmp-manager.lastEmptyCheck: 1658867012
media.gmp-widevinecdm.abi: x86_64-msvc-x64
media.gmp-widevinecdm.lastUpdate: 1652805489
media.gmp-widevinecdm.version: 4.10.2449.0
media.gmp.storage.version.observed: 1
media.hardware-video-decoding.failed: false
media.videocontrols.picture-in-picture.video-toggle.enabled: false
network.dns.disablePrefetch: true
network.http.referer.disallowCrossSiteRelaxingDefault.top_navigation: true
network.http.speculative-parallel-limit: 0
network.IDN_show_punycode: true
network.predictor.enabled: false
network.prefetch-next: false
network.trr.blocklist_cleanup_done: true
network.trr.mode: 2
network.trr.uri: https://mozilla.cloudflare-dns.com/dns-query
places.database.lastMaintenance: 1658430366
privacy.annotate_channels.strict_list.enabled: true
privacy.clearOnShutdown.downloads: false
privacy.clearOnShutdown.formdata: false
privacy.clearOnShutdown.history: false
privacy.clearOnShutdown.offlineApps: true
privacy.clearOnShutdown.sessions: false
privacy.donottrackheader.enabled: true
privacy.firstparty.isolate: true
privacy.history.custom: true
privacy.partition.network_state.ocsp_cache: true
privacy.purge_trackers.date_in_cookie_database: 0
privacy.purge_trackers.last_purge: 1658869931788
privacy.query_stripping.enabled: true
privacy.sanitize.pending: [{"id":"shutdown","itemsToClear":["cache","cookies","offlineApps"],"options":{}},{"id":"newtab-container","itemsToClear"
privacy.sanitize.sanitizeOnShutdown: true
privacy.trackingprotection.enabled: true
privacy.trackingprotection.socialtracking.enabled: true
privacy.userContext.enabled: true
privacy.userContext.extension: @contain-facebook
privacy.userContext.ui.enabled: true
security.remote_settings.crlite_filters.checked: 1656594349
security.remote_settings.intermediates.checked: 1656594349
security.sandbox.content.tempDirSuffix: {cbbe2ed9-f1b4-444d-9d36-d5098231e586}
security.sandbox.content.win32k-experiment.startupEnrollmentStatus: 2
services.sync.declinedEngines: tabs,addresses,creditcards,history
services.sync.engine.addresses.available: true
services.sync.engine.prefs.modified: false
services.sync.engine.tabs: false
services.sync.lastPing: 1658873932
services.sync.lastSync: Tue Jul 26 2022 17:28:54 GMT-0500 (Central Daylight Time)
signon.generation.enabled: false
storage.vacuum.last.index: 1
storage.vacuum.last.places.sqlite: 1656309330
ui.osk.debug.keyboardDisplayReason: IKPOS: Touch screen not found.
ui.osk.enabled: false

Important Locked Preferences
----------------------------

fission.autostart.session: true

Places Database
---------------

Accessibility
-------------

Activated: false
Prevent Accessibility: 0
Accessible Handler Used: true
Accessibility Instantiator:

Library Versions
----------------

NSPR
Expected minimum version: 4.34
Version in use: 4.34

NSS
Expected minimum version: 3.80
Version in use: 3.80

NSSSMIME
Expected minimum version: 3.80
Version in use: 3.80

NSSSSL
Expected minimum version: 3.80
Version in use: 3.80

NSSUTIL
Expected minimum version: 3.80
Version in use: 3.80

Sandbox
-------

Content Process Sandbox Level: 6
Effective Content Process Sandbox Level: 6
Win32k Lockdown State for Content Process: Win32k Lockdown enabled -- user in Treatment Group

Startup Cache
-------------

Disk Cache Path: C:\Users\YaRight\AppData\Local\Mozilla\Firefox\Profiles\rsuphcht.default-release\startupCache\startupCache.8.little
Ignore Disk Cache: false
Found Disk Cache on Init: true
Wrote to Disk Cache: true

Internationalization & Localization
-----------------------------------

Application Settings
Requested Locales: ["en-US"]
Available Locales: ["en-US"]
App Locales: ["en-US"]
Regional Preferences: ["en-US"]
Default Locale: "en-US"
Operating System
System Locales: ["en-US"]
Regional Preferences: ["en-US"]

Remote Debugging (Chromium Protocol)
------------------------------------

Accepting Connections: false
URL:

Printing
--------

Modified print settings
-----------------------

print_printer: \\AsIf\HP LaserJet 1020
print.more-settings.open: true
print.printer_\\AsIf\HP_LaserJet_1020.print_bgcolor: false
print.printer_\\AsIf\HP_LaserJet_1020.print_bgimages: false
print.printer_\\AsIf\HP_LaserJet_1020.print_duplex: 0
print.printer_\\AsIf\HP_LaserJet_1020.print_edge_bottom: 0
print.printer_\\AsIf\HP_LaserJet_1020.print_edge_left: 0
print.printer_\\AsIf\HP_LaserJet_1020.print_edge_right: 0
print.printer_\\AsIf\HP_LaserJet_1020.print_edge_top: 0
print.printer_\\AsIf\HP_LaserJet_1020.print_footercenter:
print.printer_\\AsIf\HP_LaserJet_1020.print_footerleft: &PT
print.printer_\\AsIf\HP_LaserJet_1020.print_footerright: &D
print.printer_\\AsIf\HP_LaserJet_1020.print_headercenter:
print.printer_\\AsIf\HP_LaserJet_1020.print_headerleft: &T
print.printer_\\AsIf\HP_LaserJet_1020.print_headerright: &U
print.printer_\\AsIf\HP_LaserJet_1020.print_in_color: false
print.printer_\\AsIf\HP_LaserJet_1020.print_margin_bottom: 0.5
print.printer_\\AsIf\HP_LaserJet_1020.print_margin_left: 0.5
print.printer_\\AsIf\HP_LaserJet_1020.print_margin_right: 0.5
print.printer_\\AsIf\HP_LaserJet_1020.print_margin_top: 0.5
print.printer_\\AsIf\HP_LaserJet_1020.print_orientation: 1
print.printer_\\AsIf\HP_LaserJet_1020.print_page_delay: 50
print.printer_\\AsIf\HP_LaserJet_1020.print_paper_height: 11
print.printer_\\AsIf\HP_LaserJet_1020.print_paper_id: 1
print.printer_\\AsIf\HP_LaserJet_1020.print_paper_size_unit: 0
print.printer_\\AsIf\HP_LaserJet_1020.print_paper_width: 8.5
print.printer_\\AsIf\HP_LaserJet_1020.print_resolution: 600
print.printer_\\AsIf\HP_LaserJet_1020.print_reversed: false
print.printer_\\AsIf\HP_LaserJet_1020.print_scaling: 1
print.printer_\\AsIf\HP_LaserJet_1020.print_shrink_to_fit: true
print.printer_\\AsIf\HP_LaserJet_1020.print_to_file: false
print.printer_\\AsIf\HP_LaserJet_1020.print_unwriteable_margin_bottom_twips: 230
print.printer_\\AsIf\HP_LaserJet_1020.print_unwriteable_margin_left_twips: 230
print.printer_\\AsIf\HP_LaserJet_1020.print_unwriteable_margin_right_twips: 336
print.printer_\\AsIf\HP_LaserJet_1020.print_unwriteable_margin_top_twips: 230
print.printer_Microsoft_Print_to_PDF.print_bgcolor: false
print.printer_Microsoft_Print_to_PDF.print_bgimages: false
print.printer_Microsoft_Print_to_PDF.print_duplex: 0
print.printer_Microsoft_Print_to_PDF.print_edge_bottom: 0
print.printer_Microsoft_Print_to_PDF.print_edge_left: 0
print.printer_Microsoft_Print_to_PDF.print_edge_right: 0
print.printer_Microsoft_Print_to_PDF.print_edge_top: 0
print.printer_Microsoft_Print_to_PDF.print_footercenter:
print.printer_Microsoft_Print_to_PDF.print_footerleft:
print.printer_Microsoft_Print_to_PDF.print_footerright:
print.printer_Microsoft_Print_to_PDF.print_headercenter:
print.printer_Microsoft_Print_to_PDF.print_headerleft:
print.printer_Microsoft_Print_to_PDF.print_headerright:
print.printer_Microsoft_Print_to_PDF.print_in_color: true
print.printer_Microsoft_Print_to_PDF.print_margin_bottom: 0.5
print.printer_Microsoft_Print_to_PDF.print_margin_left: 0.5
print.printer_Microsoft_Print_to_PDF.print_margin_right: 0.5
print.printer_Microsoft_Print_to_PDF.print_margin_top: 0.5
print.printer_Microsoft_Print_to_PDF.print_orientation: 0
print.printer_Microsoft_Print_to_PDF.print_page_delay: 50
print.printer_Microsoft_Print_to_PDF.print_paper_height: 11
print.printer_Microsoft_Print_to_PDF.print_paper_id: 1
print.printer_Microsoft_Print_to_PDF.print_paper_size_unit: 0
print.printer_Microsoft_Print_to_PDF.print_paper_width: 8.5
print.printer_Microsoft_Print_to_PDF.print_resolution: 600
print.printer_Microsoft_Print_to_PDF.print_reversed: false
print.printer_Microsoft_Print_to_PDF.print_scaling: 1
print.printer_Microsoft_Print_to_PDF.print_shrink_to_fit: true
print.printer_Microsoft_Print_to_PDF.print_to_file: false
print.printer_Microsoft_Print_to_PDF.print_unwriteable_margin_bottom_twips: 0
print.printer_Microsoft_Print_to_PDF.print_unwriteable_margin_left_twips: 0
print.printer_Microsoft_Print_to_PDF.print_unwriteable_margin_right_twips: 0
print.printer_Microsoft_Print_to_PDF.print_unwriteable_margin_top_twips: 0
print.printer_Mozilla_Save_to_PDF.print_bgcolor: false
print.printer_Mozilla_Save_to_PDF.print_bgimages: false
print.printer_Mozilla_Save_to_PDF.print_duplex: 0
print.printer_Mozilla_Save_to_PDF.print_edge_bottom: 0
print.printer_Mozilla_Save_to_PDF.print_edge_left: 0
print.printer_Mozilla_Save_to_PDF.print_edge_right: 0
print.printer_Mozilla_Save_to_PDF.print_edge_top: 0
print.printer_Mozilla_Save_to_PDF.print_footercenter:
print.printer_Mozilla_Save_to_PDF.print_footerleft: &PT
print.printer_Mozilla_Save_to_PDF.print_footerright: &D
print.printer_Mozilla_Save_to_PDF.print_headercenter:
print.printer_Mozilla_Save_to_PDF.print_headerleft: &T
print.printer_Mozilla_Save_to_PDF.print_headerright: &U
print.printer_Mozilla_Save_to_PDF.print_in_color: true
print.printer_Mozilla_Save_to_PDF.print_margin_bottom: 0.5
print.printer_Mozilla_Save_to_PDF.print_margin_left: 0.5
print.printer_Mozilla_Save_to_PDF.print_margin_right: 0.5
print.printer_Mozilla_Save_to_PDF.print_margin_top: 0.5
print.printer_Mozilla_Save_to_PDF.print_orientation: 0
print.printer_Mozilla_Save_to_PDF.print_page_delay: 50
print.printer_Mozilla_Save_to_PDF.print_paper_height: 11
print.printer_Mozilla_Save_to_PDF.print_paper_id: na_letter
print.printer_Mozilla_Save_to_PDF.print_paper_size_unit: 0
print.printer_Mozilla_Save_to_PDF.print_paper_width: 8.5
print.printer_Mozilla_Save_to_PDF.print_resolution: 0
print.printer_Mozilla_Save_to_PDF.print_reversed: false
print.printer_Mozilla_Save_to_PDF.print_scaling: 1
print.printer_Mozilla_Save_to_PDF.print_shrink_to_fit: true
print.printer_Mozilla_Save_to_PDF.print_to_file: true
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_bottom_twips: 0
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_left_twips: 0
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_right_twips: 0
print.printer_Mozilla_Save_to_PDF.print_unwriteable_margin_top_twips: 0
```

**Comment by dannycolin** [comment]
Duplicate of #1958

**Comment by dannycolin** [comment]
Only the list in the addon popup can be reordered. https://bugzilla.mozilla.org/show_bug.cgi?id=1333395 would need to be implemented in Firefox itself to add the ability to reorder the other menus.

**Comment by wumpus42** [comment]
> Only the list in the addon popup can be reordered. https://bugzilla.mozilla.org/show_bug.cgi?id=1333395 would need to be implemented in Firefox itself to add the ability to reorder the other menus.

Where does Firefox get the list of container names to display in the menu? Doesn't the addon create/populate that list somehow then make it available to Firefox to display? Container names are certainly removed from that list when I remove them from the toolbar list. Could the addon create the list for Firefox in the same order that the container names have been managed/ordered in the addon toolbar?

**Comment by dannycolin** [comment]
> Doesn't the addon create/populate that list somehow then make it available to Firefox to display? 

No. The addon can create/delete containers but it doesn't render the list in the Firefox UI. The only thing rendered by the addon is the content inside the popup when you click the Multi-Account Containers icon.
